### DISCUSSION TITLE
How do we use Neo4j in Sunbird? Where can i find the schema of the same?

### DISCUSSION THREAD

**Discussion by aks30** [Sunbird-coKreat/Community#7] [Q&A] [answered]
I was trying to understand how we are using a graph database and where can I find the schema for the same.

I tried looking at  - https://cokreat.sunbird.org/learn/overview thinking since the questions and courses etc are created here but couldn't find it.

Please do let me know if I should be looking in some other building block.

@pallakartheekreddy , @krgauraw @maheshkumargangula, @vijiurs

**Comment by pallakartheekreddy** [comment, marked as answer]
@aks30 `coKreat` building block using `knowlg` & `inQuiry` building block components to create courses & questions. 

`coKreat` building block maintains its own object schemas for knowlg and inQuiry components(services). Please find below for coKreat specific schemes

https://cokreat.sunbird.org/use/developer-installation/reference-cokreat-web-app

Please refer [here](https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service/schemas) for knowlg default schemas and [here](https://inquiry.sunbird.org/learn/product-and-developer-guide/question-and-question-set-service/schema) for inQuiry default schemas

**Reply by aks30** [reply]
Thanks @pallakartheekreddy !

@vinukumar-vs I believe there is some recording or session that explains the flow of data from neo4j to ES and Redis, if you can point us to that, please?

**Reply by vinukumar-vs** [reply]
@aks30 Take a look at these KT-Session of Knowledge platform
https://drive.google.com/drive/u/1/folders/1uoa7ZOc780F86cjUjVJYlVl25FrxXj-L

@Krishnaj20 Please share the drive access to the public. So that anyone can view these KT session videos.

**Reply by Krishnaj20** [reply]
@vinukumar-vs - the drive has public view access. I believe it will be useful to point to the video in the drive which can provide these details.

@aks30 - this query should be posted [here](https://github.com/Sunbird-Knowlg/Community/discussions).

**Reply by aks30** [reply]
Sure @Krishnaj20 , will do so next time

**Reply by Krishnaj20** [reply]
Dear @aks30 - we will cover this topic in detail in the first session of Knowlg community connect. @vinukumar-vs will shortly post about this in the Knowlg discussion forums.
